templates/user/anonymous_user_to_project.html

```diff
@@ -0,0 +1,252 @@
+      rel="stylesheet"
+    />
+
+    <style>
+      body {
+        font-family: "Roboto", sans-serif;
+      }
+    </style>
+  </head>
+
+  <body style="background-color: #f4f4f4; margin: 0 !important; padding: 0 !important">
+    <!-- HIDDEN PREHEADER TEXT -->
+    <div
+      style="
+        display: none;
+        font-size: 1px;
+        color: #fefefe;
+        line-height: 1px;
+        max-height: 0px;
+        max-width: 0px;
+        opacity: 0;
+        overflow: hidden;
+      "
+    ></div>
+    <table border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td bgcolor="#f4f4f4" align="center" style="padding: 0px 10px 0px 10px">
+          <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px">
+            <tr>
+              <td
+                bgcolor="#ffffff"
+                align="center"
+                valign="top"
+                style="
+                  padding: 5px 15px 30px 15px;
+                  border-radius: 4px 4px 0px 0px;
+                  color: #111111;
+                  font-family: 'Roboto', Helvetica, Arial, sans-serif;
+                  font-size: 48px;
+                  font-weight: 400;
+                  letter-spacing: 4px;
+                  line-height: 48px;
+                "
+              >
+                <img
+                  src="https://raw.githubusercontent.com/crane-cloud/frontend/develop/public/favicon.png"
+                  width="150"
+                  height="100"
+                  style="display: block; border: 0px"
+                />
+              </td>
+            </tr>
+          </table>
+        </td>
+      </tr>
+
+      <tr>
+        <td bgcolor="#f4f4f4" align="center" style="padding: 0px 10px 0px 10px">
+          <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px">
+            <tr>
+              <td
+                bgcolor="#ffffff"
+                align="left"
+                style="
+                  padding: 5px 15px 30px 15px;
+                  color: #666666;
+                  font-family: 'Roboto', Helvetica, Arial, sans-serif;
+                  font-size: 18px;
+                  font-weight: 400;
+                  line-height: 25px;
+                "
+              >
+                <p style="margin: 0">Dear User,</p>
+              </td>
+            </tr>
+            <tr>
+              {% if success ==true %}
+
+              <td
+                bgcolor="#ffffff"
+                align="left"
+                style="
+                  padding: 5px 15px 30px 15px;
+                  color: #666666;
+                  font-family: 'Roboto', Helvetica, Arial, sans-serif;
+                  font-size: 18px;
+                  font-weight: 400;
+                  line-height: 25px;
+                "
+              >
+                <p style="margin: 0">You've been assigned a role: {{email_role}} in the {{project_name}} project!</p>
```

Comment:
<img width="1044" alt="Screenshot 2022-11-15 at 12 43 21 PM" src="https://user-images.githubusercontent.com/18049550/201886326-06e9cb77-1f3a-4d97-81af-41b9826d8aff.png">
As for the email working, we can try to improve it.
For example, tell the user why there invited and also who invited them ie;

> Hey there,
> This email is to inform you that you have been invited to collaborate on project **Demo project** by `henry@cranecloud.io`....

Reply:
This has been worked on